www/layouts/about/section.html

```diff
@@ -327,7 +327,7 @@ <h1 class="text-center-mobile pb-5">Beyond Course Materials</h1>
               practicing teachers, covering topics such as course design, active 
               learning methods, and strategies to engage learners.
             </p>
-            <a class="red-btn" href="/search/?type=Instructor%20Insights&s=-runs.best_start_date"
+            <a class="red-btn" href="https://ocwnext.odl.mit.edu/search/?f=Instructor%20Insights&s=-runs.best_start_date"
```

Comment:
I think we should do it something like `href="/search/...` instead of `href="https://ocwnext.odl.mit.edu/...` as currently prod is on `https://ocw.mit.edu...`

Reply:
Yes!

Reply:
Yes I did that in my first update but the second round of updated suggested the full link as you can see in the diff so I updated it accordingly.
I will revert it back